pkg/config/config.go

```diff
@@ -1218,6 +1222,7 @@ func load(config Config, origin string, loadSecret bool) (*Warnings, error) {
 	useHostEtc(config)
 	loadProxyFromEnv(config)
 	SanitizeAPIKeyConfig(config, "api_key")
+	sanitizeExternalMetricsProviderChunkSize(config)
```

Comment:
We have the `overrideFuncs` to register functions like that

Reply:
Something like this: https://github.com/DataDog/datadog-agent/pull/10781/commits/5a2915f5acd4eae9eeb794d8448a96dcfd05fa95 ?